### PR TITLE
Implement of generate data according to the 6 variant

### PR DESCRIPTION
**Pull request by denysmk**

### CODE REVIEW

**Review by oleksandr-kychun — COMMENTED** [review]
correct and simple solution)

File: homework.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
-VARIANT = 0
+VARIANT = 6
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  rand(10..99).to_s + '-' + rand(1000000..9999999).to_s + '-' + [*"A".."Z"].sample(3).join
```
Comment (sveredyuk) [inline homework.rb]: Magic numbers.
No self-explaining variables?

Comment (sergiy-koshoviy) [reply homework.rb]: '+' - too expensive operator. Try to rewrite with interpolation

File: homework.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
-VARIANT = 0
+VARIANT = 6
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  first_part = rand(10..99).to_s
+  second_part = rand(1000000..9999999).to_s
+  third_part = [*"A".."Z"].sample(3).join
```
Comment (ruslan-369) [inline homework.rb]: `('A'..'Z').to_a` is better than `[*"A".."Z"]` in context of readability

**Review by sveredyuk — APPROVED** [review]

**Pull request closed without merging**